support/spring-data-jpa-javax/src/main/kotlin/com/linecorp/kotlinjdsl/support/spring/data/jpa/javax/repository/KotlinJdslJpqlExecutorImpl.kt

```diff
@@ -37,27 +37,33 @@ open class KotlinJdslJpqlExecutorImpl(
     private val metadata: CrudMethodMetadata?,
 ) : KotlinJdslJpqlExecutor {
     override fun <T : Any> findAll(
+        offset: Int?,
+        limit: Int?,
         init: Jpql.() -> JpqlQueryable<SelectQuery<T>>,
     ): List<T?> {
-        return findAll(Jpql, init)
+        return findAll(Jpql, offset, limit, init)
```

Comment:
Both offset and limit are Integer types, so how about specifying the parameter names? I think it would be better to include them when calling the method, just like in the example you wrote. The rest of the code is the same.

--- korean
offset, limit 모두 Integer 타입인데 파라메터의 이름을 명시하는게 어떤가요? 예제로 적어놓으신 부분과 동일하게 메소드 호출시에도 넣어두는게 좋을 것 같습니다. 나머지 코드도 동일합니다.